python/TestHarness/testers/Tester.py

```diff
@@ -70,6 +71,8 @@ def __init__(self, name, params):
   def getInputFile(self):
     return None
 
+  def getStatusMessage(self):
+    return self.specs['status_message']
```

Comment:
You have the right idea with the method here but don't use the InputParameters as writable variables. It's technically OK to do it but it actually doesn't make much sense semantically. The status message should be held in a member variable of the Tester class and you'll set that and report that variable instead (e.g. `self.status_message`).